package-lock.json

```diff
@@ -0,0 +1,14186 @@
+{
```

Comment:
Since we are just using `yarn` lets delete `npm`'s lock file.

Reply:
Yes, my bad committing that I'll fix it